theme.js

```diff
@@ -1 +1,36 @@
-import './js/blogtini.js'
+/**
+ * Dress up all the things
+ */
+
+/**
+ * What is the intented URL to expose this site, it's most likely over http, not "file://",
+ * and possibly not at the root of the domain name.
+ */
+const PRODUCTION_SITE_ROOT_BASE_URL = 'https://renoirb.github.io/blogtini/'
```

Comment:
Declare what's your intended production location.  If it's not that, it's for dev.  It's at the root (`/theme.js`) and it gets passed at the import below.